docusaurus/docs/form/upload/upload.md

```diff
@@ -54,31 +56,31 @@ Callback to be executed when file is uploaded. The callback is provided the `Upl
 
-#### `onFileRemove?: (files: Array<File>, fileId: String) => void`
+#### `onFileRemove?: (files: File[], fileId: String) => void`
 
 Callback called when file is removed. The callback is provided two arguments. 1. the updated files and 2. the id of the file that was removed
 
-### `fileDeliveryMetadata?: object | (upload: UploadCore) => object`
+#### `fileDeliveryMetadata?: object | (upload: UploadCore) => object`
 
 The metadata properties that have been configured for the delivery channel you are trying to reach with `avFilesDeliveryApi`.
 
-#### Use this prop together with `deliveryChannel` and `deliverFileOnSubmit` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
+Use this prop together with `deliveryChannel` and `deliverFileOnSubmit` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
 
 If you have a use case where some of your metadata is dependent on information from the upload, you can define `fileDeliveryMetadata` as a function that accepts an upload variable. The upload will be passed to your function before the file delivery API call, and you can assign any dynamic props that you need to before returning the metadata object. See [example File Delivery Metadata Function Usage below](###File-Delivery-Metadata-Function-Usage)
 
 ### `deliveryChannel?: string`
 
 The name of the delivery channel that is unique to where you will deliver files via the `avFileDeliveryApi`.
 
-#### Use this prop together with `fileDeliveryMetadata` and `deliverFileOnSubmit` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
+Use this prop together with `fileDeliveryMetadata` and `deliverFileOnSubmit` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
 
-### `deliverFileOnSubmit?: boolean`
+#### `deliverFileOnSubmit?: boolean`
 
 This prop is used in tandem with `deliveryChannel` and `fileDeliveryMetadata` so that your files will only get delivered to `fileDeliveryApi` when the form is being submitted. **Default:** `false`
 
-#### Use this prop together with `fileDeliveryMetadata` and `deliveryChannel` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
+Use this prop together with `fileDeliveryMetadata` and `deliveryChannel` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
```

Comment:
And last place but I think we should add the same `##### Usage with File Delivery API` header here